<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QWidget" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Prev</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_area">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_counter">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>510</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="threshold_check">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QSlider" name="threshold_slider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>254</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="threshold_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
